packages/app/src/app/pages/Sandbox/SearchDependencies/SearchBox.tsx

```diff
@@ -44,17 +44,21 @@ export const SearchBox = ({ handleManualSelect, onChange, listRef }) => {
           // do nothing
         } else {
           const prev = activeElement.previousSibling as HTMLButtonElement;
-          prev.focus();
+          if (prev?.focus) {
+            prev.focus();
+          }
         }
       }
       if (down) {
         if (activeElement === last) {
           // do nothing
-        } else if (activeElement === input) {
+        } else if (activeElement === input && first) {
           first.focus();
         } else {
           const next = activeElement.nextSibling as HTMLButtonElement;
-          next.focus();
+          if (next?.focus) {
+            next.focus();
```

Comment:
doing `next?.focus?.();` should work 🤔

Reply:
It actually still gave me an error that said that focus was not a function

So I had a next but it wasn't an element